2022q2/golang_riscv64.adoc

```diff
@@ -0,0 +1,15 @@
+=== go on FreeBSD riscv64
+
+Links: +
+link:https://github.com/golang/go[golang Home Page] URL: link:https://github.com/golang/go[https://github.com/golang/go] +
+link:https://github.com/MikaelUrankar/go/tree/freebsd_riscv64[FreeBSD riscv64 github repo] URL: link:https://github.com/MikaelUrankar/go/tree/freebsd_riscv64[https://github.com/MikaelUrankar/go/tree/freebsd_riscv64] +
+link:https://github.com/golang/go/issues/53466[FreeBSD riscv64 golang issue] URL: link:https://github.com/golang/go/issues/53466[https://github.com/golang/go/issues/53466]
+
+Contact: Mikaël Urankar <mikael@FreeBSD.org> +
+Contact: Dmitri Goutnik <dmgk@FreeBSD.org>
+
+Work has been done to port go on FreeBSD riscv64 which builds and passes all
+run.bash tests, including cgo (tested under QEMU and on Unmatched).
+A pull request is created upstream and the proposal has been added to the active
```

Comment:
I'd change "is" to "was" or "has been" '(probably the latter to match the second clause). But that's very minor so I can take care of it when merging.